jb-lib/lib/channel_handler/pinnacle_whatsapp_handler.py

```diff
@@ -0,0 +1,439 @@
+import base64
+import json
+import os
+from typing import Any, Dict, Generator
+import requests
+
+from sqlalchemy import select
+
+from lib.db_connection import sync_session
+from lib.azure_storage_sync import AzureStorageSync
+from lib.channel_handler.channel_handler import ChannelData, RestChannelHandler, User
+from lib.channel_handler.language import LanguageMapping, LanguageCodes
+from lib.data_models import (
+    MessageType,
+    Message,
+    TextMessage,
+    AudioMessage,
+    InteractiveMessage,
+    Option,
+    FormMessage,
+    ImageMessage,
+    DocumentMessage,
+    InteractiveReplyMessage,
+    RestBotInput,
+    ListMessage,
+    ButtonMessage,
+    DialogMessage,
+    DialogOption,
+    FormReplyMessage
+)
+from lib.models import JBChannel, JBUser, JBForm
+from lib.utils import EncryptionHandler
+
+azure_creds = {
```

Comment:
Use lib.file_storage instead.

Refer src/extensions.py in language

Reply:
done